Sources/RInAppMessaging/Views/Presenters/FullViewPresenter.swift

```diff
@@ -155,7 +149,7 @@ internal class FullViewPresenter: BaseViewPresenter, FullViewPresenterType, Erro
     }
 
     private func trackPushPrimerAction(didOptIn: Bool) {
-        AnalyticsBroadcaster.sendEventName(Constants.RAnalytics.pushPrimerEventName,
+        AnalyticsTracker.sendEventName(Constants.RAnalytics.pushPrimerEventName,
                                            dataObject: [Constants.RAnalytics.Keys.pushPermission: didOptIn ? 1 : 0,
```

Comment:
nit: please fix indentation in `dataObject` lines

Reply:
This is something should learn from you

Reply:
Fixed